regression/goto-instrument/replace-calls-05/main.c

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
remove blank line (same in other files)

Reply:
Ok, done (but github doesn't seem to notice).